packages/iov-lisk/src/derivation.ts

```diff
@@ -1,12 +1,37 @@
 import { Address } from "@iov/bcp";
-import { Derivation } from "@iov/dpos";
+import { Ed25519, Ed25519Keypair, Sha256 } from "@iov/crypto";
+import { Encoding } from "@iov/encoding";
+import Long from "long";
 
 import { constants } from "./constants";
 
-export function pubkeyToAddress(pubkey: Uint8Array): Address {
-  return Derivation.pubkeyToAddress(pubkey, constants.addressSuffix);
-}
+export class Derivation {
```

Comment:
It is used as a namespace (something we do a couple of times in IOV Core). This may be a discouraged thing to do in JS but then again I don't see a great alternative. I'm not a huge fan of unprefixed free functions, but yeah, no strong opinion here.